web/components/Feed/Feed.tsx

```diff
@@ -607,20 +607,26 @@ function Feed(props: {
             filter
           ) : (
             <Stack overflowY="auto" spacing={0} w="100%" mt={4}>
-              {debouncedFeedPosts?.map((p) => {
-                return (
-                  <Box
-                    onClick={() => {
-                      setModalPostId(p._id);
-                      onPostViewOpen();
-                    }}
-                    _hover={{ cursor: "pointer" }}
-                    key={p._id.toString()}
-                  >
-                    <FeedPostCard post={p} />
-                  </Box>
-                );
-              })}
+              {(debouncedFeedPosts?.length ?? 0) > 0 ? (
```

Comment:
Sorry, I think I misspoke earlier - keep the `debouncedFeedPosts && debouncedFeedPosts > 0` condition from earlier. I'm so sorry about that 😭